.github/workflows/R-CMD-check.yaml

```
@@ -4,21 +4,26 @@
+    runs-on: ${{ matrix.config.os }}
 
-    name: ${{ matrix.os }} (${{ matrix.r-version }})
+    name: ${{ matrix.config.os }} (${{ matrix.config.r }})
 
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
-        r-version: ['oldrel-1', 'release', 'devel']
+        config:
+          - {os: macos-latest,   r: 'release'}
```

yamllint

[error] too many spaces after comma

(commas)